Erupe/server/channelserver/handlers.go

```diff
@@ -356,9 +357,100 @@ func handleMsgMhfAcquireTitle(s *Session, p mhfpacket.MHFPacket) {}
 
 func handleMsgMhfEnumerateTitle(s *Session, p mhfpacket.MHFPacket) {}
 
-func handleMsgMhfEnumerateUnionItem(s *Session, p mhfpacket.MHFPacket) {}
+type Item struct {
```

Comment:
Third declaration of Item struct.

Reply:
Background conversation has been had, There was confusion between this fork and his own.  I'm going to combine both struts and move them to a common place. Wish has suggested handlers.go. for now